<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1078</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1081</width>
     <height>721</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(24, 24, 189);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>1081</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(26, 57, 255);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>10</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>520</y>
     <width>1081</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(26, 57, 255);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>10</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>30</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_transfer_balance">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>580</y>
     <width>161</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>transfer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_withdraw_balance">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>580</y>
     <width>161</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>withdraw</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_deposit_balance">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>580</y>
     <width>161</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>deposit</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../test1.qrc"/>
 </resources>
 <connections/>
</ui>
